packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -500,6 +535,11 @@ const NonBarfTurnTasks: AlternateTask[] = [
       }),
     sobriety: "sober",
   },
+  {
+    name: "CyberRealm 3",
+    ...cyberRealm3(() => !willDrunkAdventure()),
```

Comment:
you shouldn't follow this pattern for a sober-only task, and indeed you shouldn't pass `() => !willDrunkAdventure()` if you want to run this while sober even in situations where we'll drunk adventure that day